### ISSUE TITLE
The drmad loss grows to NaN

### ISSUE THREAD

**Issue by nicholas-leonard** [nicholas-leonard/drmad#3]

**Comment by bigaidream** [comment]
"Large learning rates induce chaotic behavior in the learning dynamics, making the gradient uninformative about the medium-term shape of the training objective. This phenomenon is related to the exploding-gradient problem." -- in the `4. Limitations` part of [Gradient-based hyperparameter optimization through reversible learning](http://arxiv.org/pdf/1502.03492.pdf)

The elementary learning rate was too high in my previous code. After setting it to a small value, e.g. 0.00001, no NaN anymore. 

Another solution is to stop hyper-optimization when the magnitude of the hyper-gradient begins to grow.
